bin/pubber

```diff
@@ -2,8 +2,8 @@
 
 ROOT_DIR=$(realpath $(dirname $0)/..)
 
-if [ $# != 4 ]; then
-    echo $0 SITE_PATH PROJECT_ID DEVICE_ID SERIAL_NO
+if [[ "${#}" -lt "4" ]]; then
```

Comment:
Why this?  $# is a lot easier to read!  (Also, quotes are unnecessary with [[, since it's an explicit different parser to avoid the extra clutter!)

Reply:
Changed to use `((` instead as per [style guide](https://google.github.io/styleguide/shellguide.html)